Cargo.toml

```diff
@@ -18,5 +18,5 @@ members = [
 ]
 
 [patch.crates-io]
-libp2p = { git = "https://github.com/dignifiedquire/rust-libp2p", branch = "feat-kad-count" }
+libp2p = { git = "https://github.com/arqu/rust-libp2p", branch = "feat-metrics-clone" }
```

Comment:
Did this because my branch (based of `feat-kad-count`) allows the libp2p metrics struct to be cloneable.

Reply:
@Arqu can we change this/merge this? I would love to not have more branches. Alternatively we could just move to a specific fork on n0-computer maybe that we use

Reply:
yeah lets merge it

Reply:
for whatever reason, I cant use your branch as a base (cant find it) for a PR. Can you just locally smush it in with a cherry pick

Reply:
No longer needed